crates/js-sys/tests/wasm/SharedArrayBuffer.js

```diff
@@ -0,0 +1,3 @@
+exports.is_shared_array_buffer_supported = function () {
```

Comment:
@alexcrichton There is `exports` here. Does it mean that the tests in `wasm` folder run only for `node.js`?

Reply:
Indeed yeah these tests only run in node.js